packages/core/plugins/core-plugins-server-internal/src/test_helpers/create_core_context_config_service.mock.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { IConfigService } from '@kbn/config';
+import { configServiceMock } from '@kbn/config-mocks';
+import { ByteSizeValue } from '@kbn/config-schema';
+import { fromRoot } from '@kbn/utils';
+import { duration } from 'moment';
+import { from } from 'rxjs';
+
+export const createCoreContextConfigServiceMock = (): IConfigService => {
```

Comment:
We need this mock in both `plugin_context.test.ts` and `legacy_config.test.ts` so I moved it to a test_helpers file